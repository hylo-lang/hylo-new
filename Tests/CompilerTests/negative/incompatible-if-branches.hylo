struct A { public memberwise init }
struct B { public memberwise init }

public fun main() {
  _ = if true { A() } else { B() }
}
