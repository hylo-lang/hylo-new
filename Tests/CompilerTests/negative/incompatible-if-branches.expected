negative/incompatible-if-branches.hylo:5.30-33: error: no coercion from 'B' to 'A'
  _ = if true { A() } else { B() }
                             ~~~
